grid.html

```diff
@@ -251,3 +251,44 @@ <h5>Configuration of node with JSON</h5>
     "maxSessions": 5
   }
 }</code></pre>
+
+<h3>Specifying the Port</h3>
+
+<p>The default TCP/IP port used by the hub is 4444.If another application on your computer is already using this port
```

Comment:
Please simplify the explanation, just mention how to set the port, there is no need to cover different outcomes/errors due to pre-existent machine setups, besides the returned error is self explanatory.

Reply:
Agree